### ISSUE TITLE
Not working in Windows (getting the system cannot find the file specified)

### ISSUE THREAD

**Issue by acrolink** [ClaudiuGeorgiu/Obfuscapk#7]
```
# python -m obfuscapk.cli -o RandomManifest -o Rebuild -o NewSignature -o NewAlignment H:\PROJECTS\Android\MemoApp\app\release\app-release.apk
04/01/2020 21:49:20> [ERROR][obfuscapk.tool.Apktool][decode()] Error during decoding: [WinError 2] The system cannot find the file specified
04/01/2020 21:49:20> [ERROR][obfuscapk.obfuscation][decode_apk()] Error during apk decoding: [WinError 2] The system cannot find the file specified
04/01/2020 21:49:20> [ERROR][yapsy_loaded_plugin_RandomManifest_1.random_manifest.RandomManifest][obfuscate()] Error during execution of "RandomManifest" obfuscator: [WinError 2] The system cannot find the file specified
04/01/2020 21:49:20> [CRITICAL][obfuscapk.main][perform_obfuscation()] Error during obfuscation: [WinError 2] The system cannot find the file specified
Traceback (most recent call last):
  File "C:\Opt\Obfuscapk\src\obfuscapk\main.py", line 88, in perform_obfuscation
    (obfuscator_name_to_function[obfuscator_name])(obfuscation)
  File "C:\Opt\Obfuscapk\src\obfuscapk\obfuscators\random_manifest\random_manifest.py", line 108, in obfuscate
    manifest_tree = Xml.parse(obfuscation_info.get_manifest_file(), parser=xml_parser)
  File "C:\Opt\Obfuscapk\src\obfuscapk\obfuscation.py", line 439, in get_manifest_file
    self.decode_apk()
  File "C:\Opt\Obfuscapk\src\obfuscapk\obfuscation.py", line 276, in decode_apk
    apktool.decode(self.apk_path, self._decoded_apk_path, force=True)
  File "C:\Opt\Obfuscapk\src\obfuscapk\tool.py", line 61, in decode
    output = subprocess.check_output(decode_cmd, stderr=subprocess.STDOUT).strip()
  File "C:\Users\User\AppData\Local\Programs\Python\Python37\lib\subprocess.py", line 411, in check_output
    **kwargs).stdout
  File "C:\Users\User\AppData\Local\Programs\Python\Python37\lib\subprocess.py", line 488, in run
    with Popen(*popenargs, **kwargs) as process:
  File "C:\Users\User\AppData\Local\Programs\Python\Python37\lib\subprocess.py", line 800, in __init__
    restore_signals, start_new_session)
  File "C:\Users\User\AppData\Local\Programs\Python\Python37\lib\subprocess.py", line 1207, in _execute_child
    startupinfo)
FileNotFoundError: [WinError 2] The system cannot find the file specified
Traceback (most recent call last):
  File "C:\Users\User\AppData\Local\Programs\Python\Python37\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Users\User\AppData\Local\Programs\Python\Python37\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Opt\Obfuscapk\src\obfuscapk\cli.py", line 81, in <module>
    arguments.interactive, arguments.ignore_libs, arguments.virus_total_key)
  File "C:\Opt\Obfuscapk\src\obfuscapk\main.py", line 88, in perform_obfuscation
    (obfuscator_name_to_function[obfuscator_name])(obfuscation)
  File "C:\Opt\Obfuscapk\src\obfuscapk\obfuscators\random_manifest\random_manifest.py", line 108, in obfuscate
    manifest_tree = Xml.parse(obfuscation_info.get_manifest_file(), parser=xml_parser)
  File "C:\Opt\Obfuscapk\src\obfuscapk\obfuscation.py", line 439, in get_manifest_file
    self.decode_apk()
  File "C:\Opt\Obfuscapk\src\obfuscapk\obfuscation.py", line 276, in decode_apk
    apktool.decode(self.apk_path, self._decoded_apk_path, force=True)
  File "C:\Opt\Obfuscapk\src\obfuscapk\tool.py", line 61, in decode
    output = subprocess.check_output(decode_cmd, stderr=subprocess.STDOUT).strip()
  File "C:\Users\User\AppData\Local\Programs\Python\Python37\lib\subprocess.py", line 411, in check_output
    **kwargs).stdout
  File "C:\Users\User\AppData\Local\Programs\Python\Python37\lib\subprocess.py", line 488, in run
    with Popen(*popenargs, **kwargs) as process:
  File "C:\Users\User\AppData\Local\Programs\Python\Python37\lib\subprocess.py", line 800, in __init__
    restore_signals, start_new_session)
  File "C:\Users\User\AppData\Local\Programs\Python\Python37\lib\subprocess.py", line 1207, in _execute_child
    startupinfo)
FileNotFoundError: [WinError 2] The system cannot find the file specified
```

**Comment by MahariAndroid** [comment]
Please change the  line 61 , 93  130   with this line 
output = subprocess.check_output(sign_cmd, stderr=subprocess.STDOUT,**shell=True**).strip()

**Comment by ClaudiuGeorgiu** [comment]
This error should be solved with latest commit 97cb05e19fa6b5635f96083b48639de157115217.
